### ISSUE TITLE
Rename gameOf -> collaborationOf?

### ISSUE THREAD

**Issue by cdsmith** [google/codeworld#428]
I'll use this issue to consider renaming `gameOf` to `collaborationOf`.

Reasons for the change:

1. `gameOf` falsely suggests that the only types of programs one may write in the model are games.  This isn't true, though.  `gameOf` is entirely sufficient to build multi-user creative and educational activities, projects for school classes, communication software, and plenty more.
2. `gameOf` doesn't capture the key notion of the entry point.  The key notion is that multiple users are doing something together.  But "game" doesn't even suggest that.  Particularly with the popularity of mobile games, multi-player games are still the exception rather than the rule.  Note that in gloss, "game" and "play" are used to mean single-player games.
3. `collaborationOf` is more consistent in tone with other CodeWorld entry points.

@nomeata likely has some opinions here!

**Comment by nomeata** [comment]
I agree that `gameOf` is a misnomer. But so is `collaborationOf`: When we play Tic-Tac-Toe or Doom, we are certainly not collaborating.

`multiPartyInteractionOf` nails it better, but sounds horrible.

Maybe (although a bit punny) `interactionsOf`?

Other random ideas: `communicationsOf`, `onlineInteractionOf`

**Comment by cdsmith** [comment]
Interesting.  I actually thought it was pretty consistent to refer to several people playing a game together as a collaboration.  I suppose because I'm thinking of the various *programs* collaborating, even though the players are potentially in fierce competition.  I think I still prefer that to alternatives I've seen.

As long as we're brainstorming, I'll throw out `participationOf` as an option.  But I don't like it off-hand.
